courses/gnatstudio/getting_started_lab_1.rst

```diff
@@ -1,39 +0,0 @@
-.. role:: ada(code)
-   :language: ada
-
----------------
-Getting Started
----------------
-
-1. Start :toolname:`GNAT Studio`
-2. Select "Open Project"
-
-.. image:: gnat_studio/open_project/windows_1.png
```

Comment:
The image is not used anymore. Remove it.

Reply:
Several cases:

images/gnat_studio/open_project/windows_1.png
images/gnat_studio/open_project/windows_3.png
images/gnat_studio/open_project/windows_2.png
images/gnat_studio/compile/success.png
images/gnat_studio/debug_project/4.png
images/gnat_studio/ui/debug_buttons_labels.png
images/gnat_studio/ui/menu_goto_full_declaration_1.png
images/gnat_studio/ui/clean_button.png
images/gnat_studio/ui/menu_goto_full_declaration_2.png
images/gnat_studio/ui/debug_button.png
images/gnat_studio/ui/debug_perspective.png
images/gnat_studio/ui/debug_variables_view.png

Reply:
Took me a while to figure out WHY they were all unused - they've been moved to "quickstart" subfolder and redone as JPG with GNAT studio (instead of PNG with GPS)